src/pages/billing/List.js

```diff
@@ -230,30 +230,38 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
     Utils.withBusyState(setIsLoadingProjects)
   )(async () => setBillingProjects(_.sortBy('projectName', await Ajax(signal).Billing.listProjects())))
 
-  const authorizeAndLoadAccounts = _.flow(
+  const reloadBillingProject = _.flow(
+    withErrorReporting('Error loading billing project'),
+    Utils.withBusyState(setIsLoadingProjects)
+  )(async ({ projectName }) => {
+    // evaluate first to error if project doesn't exist/user can't access
+    const project = await Ajax(signal).Billing.billingProject(selectedName)
+    const index = _.findIndex({ projectName }, billingProjects)
+    setBillingProjects(_.set([index], project))
+  })
+
+  const authorizeAccounts = _.flow(
```

Comment:
`authorizeAndLoadAccounts` is being used by the `ProjectDetail` component as well - is that component expecting the accounts to be loaded when calling this function?

Reply:
I'm not sure. I assume it's left to which ever child component first needs it to call it.